src/js/_enqueues/wp/updates.js

```diff
@@ -739,34 +739,57 @@
 		if ( response.activateUrl ) {
 			setTimeout( function() {
 
-				// Transform the 'Install' button into an 'Activate' button.
-				$message.removeClass( 'install-now installed button-disabled updated-message' )
-					.addClass( 'activate-now button-primary' )
-					.attr( 'href', response.activateUrl );
-
-				if ( 'plugins-network' === pagenow ) {
-					$message
-						.attr(
-							'aria-label',
-							sprintf(
-								/* translators: %s: Plugin name. */
-								_x( 'Network Activate %s', 'plugin' ),
-								response.pluginName
-							)
-						)
-						.text( __( 'Network Activate' ) );
-				} else {
-					$message
-						.attr(
-							'aria-label',
-							sprintf(
-								/* translators: %s: Plugin name. */
-								_x( 'Activate %s', 'plugin' ),
-								response.pluginName
-							)
-						)
-						.text( __( 'Activate' ) );
-				}
+				wp.updates.ajax(
+					'check_plugin_dependencies',
+					{
+							slug: response.slug,
+							success: function() {
+								// Transform the 'Install' button into an 'Activate' button.
+								$message.removeClass( 'install-now installed button-disabled updated-message' )
+								.addClass( 'activate-now button-primary' )
+								.attr( 'href', response.activateUrl );
+
+								if ( 'plugins-network' === pagenow ) {
+									$message
+									.attr(
+										'aria-label',
+										sprintf(
+										/* translators: %s: Plugin name. */
+											_x( 'Network Activate %s', 'plugin' ),
```

Comment:
```suggestion
											/* translators: %s: Plugin name. */
											_x( 'Network Activate %s', 'plugin' ),
```
Alignment.

🔢 Applies to other places.